### DISCUSSION TITLE
Hero header breadcrumb isn't clickable?

### DISCUSSION THREAD

**Discussion by p-m-p** [Legal-and-General/canopy#154] [Q&A] [answered]
Is this by design? The breadcrumb has a transparent overlay that prevents you from clicking on the breadcrumb items and has been flagged by our QA team.

Thanks

**Comment by andy-polhill** [comment]
From having a quick look I'm pretty sure thats a bug. @aaronBery / @owensgit are you able to confirm at all? I think you may use these components.

**Reply by andy-polhill** [reply]
It looks like the overlay just needs `pointer-events: none`, it's odd how that hasn't come up before.

**Comment by aaronBery** [comment]
@thatguynamedandy  @p-m-p  we are using the hero with breadcrumb in our consuming application but because we are using a legacy grid (which sets the grid columns to a relative position) we haven't stumbled across this bug. Validating this against the Canopy Storybook example it is evident that setting the 'lg-col' classes to a relative position fixes the issue. However this is not necessarily the solution and we need to investigate to understand what the best approach for the fix is.

**Comment by aaronBery** [comment, marked as answer]
@p-m-p  I've created a issue from this https://github.com/Legal-and-General/canopy/issues/158 and submitted a PR which should fix the issue
https://github.com/Legal-and-General/canopy/pull/159

**Reply by p-m-p** [reply]
Thanks very much for jumping on this one :+1:
